task_12/style/sass/content.sass

```diff
@@ -0,0 +1,80 @@
+.main
+    background: #fff
+    width: 100% 
+    max-width: 1160px
+    margin: 0 auto
+    padding: 2em
+    filter: drop-shadow(4px 5px 6px rgba(0, 0, 0, 0.2))
+    &__wrapper
```

Comment:
You might not always need nesting. Read the thread about it [here](https://github.com/kottans-frontend-2018/assignments_iammiro/pull/2#pullrequestreview-92060771).

Reply:
Less nested code really seems more readable and shorter. Thanks again! Fixed